pkg/app/piped/livestatestore/cloudrun/store.go

```diff
@@ -34,7 +34,7 @@ type Store struct {
 type Getter interface {
 }
 
-func NewStore(cfg *config.CloudProviderCloudRunConfig, cloudProvider string, appLister applicationLister, logger *zap.Logger) *Store {
+func NewStore(ctx context.Context, cfg *config.CloudProviderCloudRunConfig, cloudProvider string, appLister applicationLister, logger *zap.Logger) *Store {
```

Comment:
Wait, why does this note exist 👀  `cfg` really not being used?

Reply:
It has been pointed out because the `NewStore` implementation has not been done yet.
[This PR](https://github.com/pipe-cd/pipecd/pull/3114) will resolve this issue.

Reply:
I see, thanks for explaination 🙏